shapely/tests/geometry/test_equality.py

```diff
@@ -232,16 +235,16 @@ def test_comparison_not_supported():
     geom2 = Point(2, 2)
 
     with pytest.raises(TypeError, match="not supported between instances"):
-        geom1 > geom2
+        assert geom1 > geom2
 
     with pytest.raises(TypeError, match="not supported between instances"):
-        geom1 < geom2
+        assert geom1 < geom2
 
     with pytest.raises(TypeError, match="not supported between instances"):
-        geom1 >= geom2
+        assert geom1 >= geom2
 
     with pytest.raises(TypeError, match="not supported between instances"):
-        geom1 <= geom2
+        assert geom1 <= geom2
```

Comment:
Plenty of good fixes like this in here.

Reply:
Why is this actually needed? We should never get to that `assert`, as the code is meant to raise an error (I don't think I ever add explicit asserts when using `with pytest.raises` contexts)

Reply:
Not really needed in this specific instance, but in most cases this is a bug, so ruff gives a linting error on it...